src/main/java/uk/gov/companieshouse/api/accounts/service/impl/DebtorsService.java

```diff
@@ -132,7 +132,7 @@ public DebtorsService (DebtorsRepository repository, DebtorsTransformer transfor
         } catch (MongoException e) {
             final Map<String, Object> debugMap = new HashMap<>();
             debugMap.put("id", id);
-            DataException dataException = new DataException("Failed to find Debtors", e);
+            DataException dataException = new DataException("Failed to find debtors", e);
```

Comment:
Should this have changed?

Reply:
No, it was an IDE update, good catch. Having looked at the class I've updated the message to use `ResourceName.DEBTORS.getName()` for consistency.